scraper.rb

```diff
@@ -141,7 +151,7 @@ def state
   SearchPage.new(url).to_h[:members]
 end.uniq
 
-members.select { |m| m[:name].start_with? 'Hon' }.each do |mem|
+members.each do |mem|
   person = MemberPage.new(mem[:url])
-  ScraperWiki.save_sqlite(%i(id name term), person.to_h)
+  ScraperWiki.save_sqlite(%i(id name term chamber), person.to_h)
```

Comment:
No, that's fine. Just making I wasn't misunderstanding something about how the site worked.